src/main/scala/org/broadinstitute/hail/variant/VariantSampleMatrix.scala

```diff
@@ -598,6 +599,76 @@ class VariantSampleMatrix[T](val metadata: VariantMetadata,
     */
   }
 
+  def aggregateByKey(keyCond: String, aggCond: String): KeyTable = {
+    val aggregationEC = EvalContext(Map(
+      "v" -> (0, TVariant),
+      "va" -> (1, vaSignature),
+      "s" -> (2, TSample),
+      "sa" -> (3, saSignature),
+      "global" -> (4, globalSignature),
+      "g" -> (5, TGenotype)))
```

Comment:
This is correct now.  It is different from other places since we're mapping per-genotype instead of per-variant or per-sample with aggregables as in other places.